interfaces/connection.go

```diff
@@ -45,6 +46,39 @@ type ConnectedSlot struct {
 	dynamicAttrs map[string]interface{}
 }
 
+// AttrGetter is an interface with Attr getter method common
+// to ConnectedSlot, ConnectedPlug, PlugInfo and SlotInfo types.
+type AttrGetter interface {
```

Comment:
This should probably be Attrer or similar. Given typical conventions, it feels like an AttrGet method should exist.

That said, this looks like a left over. There's no usage of the interface in the whole PR, so I suggest just dropping it.

Reply:
_Attrer_, did you actually mean that? How about _AttrReader_?

The _Attr_ method name needs to match the method of ConnectedPlug/Slot and PlugInfo/SlotInfo.
This interface is used in the followup branch, I just forgot to update it to match the changes to _Attr_ methods. Fixed now.

Reply:
I think Gustavo did meant Attrer, I can see at least an example where in the stdlib they went for such blunt/ungrammatical naming  https://golang.org/pkg/database/sql/driver/#Valuer

Reply:
I see. Ok, renamed.